frontend/tests/Utils.php

```diff
@@ -97,15 +99,48 @@ public static function gradeRun(
             "\x1f\x8b\x08\x08\xaa\x31\x34\x5c\x00\x03\x66\x6f" .
             "\x6f\x00\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00"
         );
-        // An empty zip file.
+        // Creating the zip file.
+        $filesContentsAsString = self::createZipFileWithContent(
+            $outputFilesContent
+        );
+
         \OmegaUp\Grader::getInstance()->setGraderResourceForTesting(
             $run,
             'files.zip',
-            "\x50\x4b\x05\x06\x00\x00\x00\x00\x00\x00\x00\x00" .
-            "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+            $filesContentsAsString
         );
     }
 
+    /**
+     * @param array<string, string>|null $filesContents
```

Comment:
```suggestion
     * @param array<string, string> $filesContents
```

es mejor que esta función siempre se llame con un arreglo y no con `null`. la cadena default se puede insertar en la otra función.

Reply:
Listo